### DISCUSSION TITLE
[코드공방]설정 화면을 만들어 보시죠!

### DISCUSSION THREAD

**Discussion by hamsik22** [M1zz/Life-iOS#19] [Show and Talk(Team Build)]
# 설정 화면을 만들어보자!

매주 토요일 1시에 모여서 각자가 만든 상황을 공유하고 있어요!
이 디스커션은 작업 중 서로의 상황을 공유하는 공간입니다!

관심 있으신 분은 언제든지 코멘트를 달아주시거나
토요일 1시 스위프트 코딩클럽을 찾아주세요!

@hamsik22 @M1zz

**Comment by Minny27** [comment]
제어센터 화면 구현 중에 '+', '-' 버튼 애니메이션 구현 중에 모르는 부분 질문드립니다.
리스트에서 값을 삭제할 때 특정 값이 삭제는 되는데, 애니메이션은 맨 끝에 있는 값에서 삭제되는 것처럼 보이는 경우가 있습니다.
보통 애니메이션을 어떻게 구현하는지 궁금합니다.
구현 화면은 다음과 같습니다.
<img src="https://user-images.githubusercontent.com/68800789/225834705-afa8ddff-08b4-49a8-baa2-192165716065.gif" width=25%>

코드는 다음과 같습니다.
```swift
func removeItemOfMyControlList(at index: Int) {
    withAnimation {
        if !myControlList.isEmpty {
            _ = myControlList.remove(at: index)
        }
    }
}

func appendToMyControlList(item: MyControl) {
    withAnimation {
        myControlList.append(item)
    }
}
